beard/clustering/wrappers.py

```diff
@@ -151,8 +170,28 @@ def fit(self, X, y=None):
                                       criterion=self.criterion,
                                       depth=self.depth, R=self.R,
                                       monocrit=self.monocrit)
+                if ground_truth:
+                    train = (y != -1)
+
+                    if train.sum() == 0:
```

Comment:
I will remove this clause as we already have this check embeded in ground_truth condition.